catroid/src/org/catrobat/catroid/content/Look.java

```diff
@@ -254,6 +255,10 @@ public LookData getLookData() {
 	public void setLookData(LookData lookData) {
 		this.lookData = lookData;
 		imageChanged = true;
+
+		if (this.equals(ProjectManager.getInstance().getCurrentProject().getSpriteList().get(0).look)) {
```

Comment:
I am not hundred Percent sure, but maybe you can check this with `if (this.getZindex() == 0)` (not sure what method, but I think you get the point). This should normally work, but you should first check if the MoveBackNSteps Brick would break that functionality. Please ask if unclear ;)